### PR TITLE
[Issue 12] Spam Handling

### PR DESCRIPTION
**Pull request by mauricerkelly**
In response to Issue #12 and in light of events this last week, I've written up some stuff about what we consider spam and how we will deal with it.

### CODE REVIEW

**Review by MarkXA — APPROVED** [review]

**Review by clairebones — APPROVED** [review]

**Comment by mauricerkelly** [comment]
@andrewbolster I'd feel happier about merging this with your approval, since you raised the issue that it was based on.

File: code-of-conduct.md
Comment (andrewbolster) [inline code-of-conduct.md]: could do with an addition of 'within a timely fashion' to the "Failure to rectify" phrase.  Other than that LGTM

**Review by andrewbolster — CHANGES_REQUESTED** [review]

**Comment by mauricerkelly** [comment]
@andrewbolster requested change has been carried out.

**Comment by mauricerkelly** [comment]
@andrewbolster still waiting on a re-review from you. Cheers.

**Review by andrewbolster — APPROVED** [review]

**Pull request merged**
